nano/node/active_transactions.hpp

```diff
@@ -137,8 +150,18 @@ class active_transactions final
 	std::condition_variable condition;
 	bool started{ false };
 	std::atomic<bool> stopped{ false };
-	static size_t constexpr confirmed_frontiers_max_pending_cut_off = 100;
+	// clang-format off
+	std::function<bool (nano::cementable_account const &, nano::cementable_account const &)> comp = [] (nano::cementable_account const & lhs, nano::cementable_account const & rhs) {
+		return lhs.blocks_uncemented < rhs.blocks_uncemented;
+	};
+	// clang-format on
+	void prioritize_frontiers_for_confirmation (std::chrono::milliseconds);
+	std::set<nano::cementable_account, decltype (comp), boost::fast_pool_allocator<nano::cementable_account>> priority_cementable_frontiers{ comp };
+	static size_t constexpr max_priority_cementable_frontiers{ 100000 };
+	static size_t constexpr confirmed_frontiers_max_pending_cut_off{ 1000 };
 	boost::thread thread;
+
+	friend class confirmation_height_prioritize_frontiers_Test;
```

Comment:
Is it used somewhere?

Reply:
The test calls the private method `prioritize_frontiers_for_confirmation ()` of this class, so either the test is a friend or that method is made public. But it is only used by the test so I thought it would be more encapsulated this way.